src/client/kotlin/tech/thatgravyboat/skyblockapi/api/events/info/TabListHeaderFooterChangeEvent.kt

```diff
@@ -2,10 +2,19 @@ package tech.thatgravyboat.skyblockapi.api.events.info
 
 import net.minecraft.network.chat.Component
 import tech.thatgravyboat.skyblockapi.api.events.base.SkyBlockEvent
+import tech.thatgravyboat.skyblockapi.utils.extentions.chunked
+import tech.thatgravyboat.skyblockapi.utils.extentions.stripColorAndS
 
 data class TabListHeaderFooterChangeEvent(
     val oldFooter: Component,
     val oldHeader: Component,
     val newFooter: Component,
     val newHeader: Component,
-) : SkyBlockEvent()
+) : SkyBlockEvent() {
+    val newFooterChunked = newFooter.chunk()
```

Comment:
while I normally dont think lazys are useful for a lot of things, here itd be useful